core/domain/feedback_services.py

```diff
@@ -115,8 +118,16 @@ def create_message(
             thread.subject = updated_subject
     thread.put()
 
+    if (user_services.is_user_registered(author_id) and len(text) > 0 and
```

Comment:
Okay, so I checked this and we do not send any emails for suggestion. Only feedback messages are being sent in email.